src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -0,0 +1,743 @@
+# This script is used as extension of sonic_yang class. It has methods of
+# class sonic_yang. A separate file is used to avoid a single large file.
+
+import yang as ly
+import re
+import syslog
+
+from json import dump, dumps, loads
+from xmltodict import parse
+from glob import glob
+
+# class sonic_yang methods, use mixin to extend sonic_yang
+class sonic_yang_ext_mixin:
```

Comment:
what is the mixin?

Reply:
https://www.linuxjournal.com/article/4540

Python mixin is way to declare a mixin class, which will extend another class. Here I extend sonic_yang class using sonic_yang_ext_mixin class. Python mixin class does not have __init__ function, that is how it differentiates b/w inheritance and mixin. I put mixin suffix on purpose to not confuse it with inheritance.